bin/prettier.js

```diff
@@ -44,21 +45,32 @@ if (!filenames.length && !stdin) {
   process.exit(1);
 }
 
+const options = {
+  printWidth: argv["print-width"],
+  tabWidth: argv["tab-width"],
+  bracketSpacing: argv["bracket-spacing"],
+  useFlowParser: argv["flow-parser"],
+  singleQuote: argv["single-quote"],
+  trailingComma: argv["trailing-comma"]
+};
+
 function format(input) {
-  return jscodefmt.format(input, {
-    printWidth: argv["print-width"],
-    tabWidth: argv["tab-width"],
-    bracketSpacing: argv["bracket-spacing"],
-    useFlowParser: argv["flow-parser"],
-    singleQuote: argv["single-quote"],
-    trailingComma: argv["trailing-comma"]
-  });
+  return jscodefmt.format(input, options);
+}
+
+function formatInput(input) {
+  if (argv["debug-print-docs"]) {
+    const docs = jscodefmt.getDocs(input, options);
```

Comment:
nit: this is actually a single "document" that is composed of others, so I'd prefer a name like `printToDocument` or something. Also makes it a little clear that it does do the first printing pass.